.goreleaser.yml

```diff
@@ -42,50 +40,6 @@ builds:
       pre: build/windows/set_exe_properties.sh {{ .Env.TAG }} "prometheus"
 
 nfpms:
```

Comment:
shoudl it be nfpms: {} or remove the line?

Reply:
Definitely, good catch.